configs/dev-config.yaml

```diff
@@ -66,3 +66,10 @@ auth:
   min_since_login: 100
 
 frontend_domain: http://localhost:3000
+
+s3:
+  allowed_types:
+    - asset
```

Comment:
Are these the special types for AWS S3? Or the types of objects we have in the postgres Assets table?